prestart.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo Prebuilding Dependent Apps...
```

Comment:
Can you add a `pushd .` and `popd` so that it doesn't change the starting dir of the user?